codec-http/src/main/java/io/netty/handler/codec/http/HttpClientCodec.java

```diff
@@ -42,7 +42,7 @@
  *
  * @see HttpServerCodec
  */
-public final class HttpClientCodec extends CombinedChannelDuplexHandler<HttpResponseDecoder, HttpRequestEncoder>
+public class HttpClientCodec extends CombinedChannelDuplexHandler<HttpResponseDecoder, HttpRequestEncoder>
```

Comment:
I am a bit hesitate to do this... Can you explain what exactly you do when you extend this ?

Reply:
Thanks for pushing back - our usage is here

https://github.com/line/armeria/blob/master/core/src/main/java/com/linecorp/armeria/client/HttpClientPipelineConfigurator.java#L634

But this line of code is positively ancient (almost 5 years old). I just tried removing it and didn't find any test failure and suspect we don't actually need to extend the codec. Removed that change.